docs/BreakingChanges/065 Corner case WPF bug in which databound text box fa.md

```diff
@@ -0,0 +1,30 @@
+## 65: Corner case WPF bug in which databound text box fails to update is fixed
```

Comment:
{
  "Id": "65",
  "Title": "Corner case WPF bug in which databound text box fails to update is fixed",
  "Details": "TODO",
  "Suggestion": "TODO",
  "Link": "https://social.msdn.microsoft.com/Forums/vstudio/en-US/c05a8c02-de67-47a9-b4ed-fd8b622a7e4a/if-i-target-net-40-but-run-on-a-machine-that-has-net-45-will-net-40-wpf-bugs-still-be-there?forum=wpf",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis?"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "Not too worried about this since it's unlikely anyone was depending on the 4.0 behavior. It's an interesting case if someone's co-developing 4.0 and 4.5, though.\nSource analyzer status: Not planned",
  "IsRetargeting": false,
  "ImpactScope": 4
}

**Parsed APIs:**
- [ ] Not detectable via API analysis?